src/meta/src/rpc/metrics.rs

```diff
@@ -34,6 +34,13 @@ pub struct MetaMetrics {
     /// latency of each barrier
     pub barrier_latency: Histogram,
 
+    /// latency between each barrier send
+    pub barrier_send_latency: Histogram,
+    /// the nums of all barrier
```

Comment:
Better to explain this in more detail. All barrier includes pending barrier on meta?

Reply:
And when will all_barrier > in-flight barrier?

Reply:
yes all_barrier = in-flight barrier + complete barrier(wait commit)